migrator/src/main/java/com/redhat/lightblue/migrator/Controller.java

```diff
@@ -224,15 +224,10 @@ public void run() {
                 MigrationConfiguration[] cfg=getMigrationConfigurations();
                 createControllers(cfg);
                 Breakpoint.checkpoint("Controller:createconfig");
-            } catch (Exception e) {
+                Thread.sleep(30000);
+            } catch (Throwable e) {
```

Comment:
Why catch the InterruptedException here? There is no need to log it as it simply means the thread has been terminated.